docs/languages/csharp/expression-trees-summary.md

```diff
@@ -0,0 +1,36 @@
+[Previous -- Translating Expressions](expression-trees-translating.md)
+
+# Summary
+
+In this series, you've seen how you can use *expression trees* to
+create dynamic programs that interpret code as data and build
+new functionality based on that code.
+
+You can examine expression trees to understand the intent of
+an algorithm. You can not only examine that code, you can modify it
+at runtime. (Actually, you can't modify an expression tree, but you
```

Comment:
I'd skip the parenthetic and make this the key point.  You can generate a new expression tree that represents modified code.